Docs/Book/Cookbook.rst

```diff
@@ -788,6 +786,130 @@ The variable `clusters` contains the results::
 
 That cluster contains 7 points, the centroid is point 6164.
 
+
+RMSD Calculation between N molecules
+------------------------------------
+
+Introduction
+````````````
+
+We sometimes need to calculate RMSD distances between two (or more) molecules. This can be used to calculate how close two conformers are. Most RMSD calculations make sense only on similar compounds or, at least, for common parts in different compounds.
+
+Details
+```````
+
+The following program (written in python 2.7) takes an SDF file as an input and generates all the RMSD distances between the molecules in that file. These distances are written to an output file (user defined).
+
+So for an SDF with 5 conformers we will get 10 RMSD scores - typical n choose k problem, without repetition i.e. 5! / 2!(5-2)!
+
+.. testcode::
```

Comment:
I think this shouldn't use the `.. testcode::` marker since it's not actually being tested. It's probably better to just use `The code:` here